packages/babel-core/test/fixtures/transformation/misc/regression-1155/output.js

```diff
@@ -1,20 +1,15 @@
 var Foo = /*#__PURE__*/function (_Bar) {
-  "use strict";
-
-  babelHelpers.inherits(Foo, _Bar);
-
-  var _super = babelHelpers.createSuper(Foo);
-
-  function Foo(options) {
-    babelHelpers.classCallCheck(this, Foo);
-    var parentOptions = {};
-
-    parentOptions.init = function () {
-      this;
-    };
-
-    return _super.call(this, parentOptions);
-  }
-
-  return babelHelpers.createClass(Foo);
-}(Bar);
+    "use strict";
```

Comment:
This is looks very unusual, the widely used convention is that it should be indented only by 1 step:
```js
var f = function () {
  something;
};
```
however, it's true that with multiple functions it looks better by indenting more:
```js
var f = function () {
    something;
  },
  g = function () {
    something;
  };
```

could we keep the original indentation if `node.declarators.length === 1`, and only use the more indented version when there are multiple declarators?